### ISSUE TITLE
301 from S3 bucket during training on AWS

### ISSUE THREAD

**Issue by willgraf** [vanvalenlab/kiosk-training#5]
Training fails on AWS with the following in STDOUT:

---

```
2018-11-27 00:02:05.222326: E tensorflow/core/platform/s3/aws_logging.cc:60] No response body. Response code: 301
2018-11-27 00:02:05.222367: W tensorflow/core/platform/s3/aws_logging.cc:57] If the signature check failed. This could be because of a time skew. Attempting to adjust the signer.
2018-11-27 00:02:05.222422: I tensorflow/core/platform/s3/aws_logging.cc:54] Found secret key
2018-11-27 00:02:05.222498: I tensorflow/core/platform/s3/aws_logging.cc:54] Connection has been released. Continuing.
2018-11-27 00:02:05.224404: W tensorflow/core/platform/s3/aws_logging.cc:57]
```

And then finally the error:

```
InternalError: : No response body. Response code: 301
InternalError: : No response body. Response code: 301
```

**Comment by willgraf** [comment]
This blocks vanvalenlab/deepcell-tf#113

Cannot test AWS training until this 301 redirect code has been figured out.
Cursory searches indicate that the URL scheme used for the `LOG_DIR` may be the culprit

**Comment by willgraf** [comment]
Looks like this is resolved by vanvalenlab/kiosk#92.

The environment variable `AWS_REGION` was set to the non-default value of `us-west-2`.  Despite the AWS nodes being deployed in that region, the bucket was expecting a region value of `us-east-1` to be able to use the `s3://bucket/key` for tensorboard's LOG_DIR.
